sickbeard/traktChecker.py

```diff
@@ -59,8 +60,7 @@ def setEpisodeToWanted(show, s, e):
         logger.log(u"Starting backlog search for %s S%02dE%02d because some episodes were set to wanted" % (show.name, s, e))
 
 
-class TraktChecker():
-
+class TraktChecker:
```

Comment:
this should be `class TracktChecker(object)`

We cant use old style classes and be compliant with both py2 and py3 (ultimate goal)